docs/sources/whatsnew/whats-new-in-v10-0.md

```diff
@@ -0,0 +1,359 @@
+---
+description: Feature and improvement highlights for Grafana v10.0
+keywords:
+  - grafana
+  - new
+  - documentation
+  - '10.0'
+  - release notes
+title: What's new in Grafana v10.0
+weight: -37
+---
+
+# What’s new in Grafana v10.0
+
+Welcome to Grafana 10.0! Read on to learn about changes to search and navigation, dashboards and visualizations, and authentication and security. For even more detail about all the changes in this release, refer to the [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
+
+<!-- Template below
+## Feature
+<!-- Name of contributor -->
+<!-- [Generally available | Available in private/public preview | Experimental] in Grafana [Open Source, Enterprise, Cloud Free, Cloud Pro, Cloud Advanced]
+Description. Include an overview of the feature and problem it solves, and where to learn more (like a link to the docs).
+> **Note:** You must use relative references when linking to docs within the Grafana repo. Please do not use absolute URLs. For more information about relrefs, refer to [Links and references](/docs/writers-toolkit/writing-guide/references/).-->
+
+## Authentication and authorization
+
+### No longer possible to disable Role-based access control
+
+<!-- Vardan Torosyan -->
+
+_Generally available in all editions of Grafana._
+
+It is no longer possible to disable [Role-based access control (RBAC)](({{< relref "../administration/roles-and-permissions/access-control/" >}})) using a configuration option in Grafana.
+
+We understand that this may affect some users who have relied on the ability to disable RBAC in the past. However, we believe that this change is necessary to ensure the best possible security and user experience for our community and customers.
+
+If you have disabled RBAC for you Grafana instance, refer to our [Upgrade Guide]({{< relref "../upgrade-guide/upgrade-v10.0/index.md" >}}) to check if the change is impacting you and what you can do to mitigate any potential issues.
+
+### SAML UI
+
+<!-- Vardan Torosyan -->
+
+_Generally available in Grafana Enterprise, Cloud Pro, and Cloud Advanced._
+
+You can now configure SAML using our new user interface, making the process easier and more convenient than ever before.
+With the new user interface, you can now configure SAML without needing to restart Grafana and you can control access to the configuration UI by using [role-based access control (RBAC)]({{< relref "../administration/roles-and-permissions/access-control/" >}}). which makes the process much faster and more efficient.
+
+The SAML UI is available in Grafana Enterprise and Grafana Cloud Pro and Advanced. It is intuitive and user-friendly, with clear instructions and helpful prompts to guide you through the process.
```

Comment:
@vtorosyan what do you think of telling users how to navigate to the UI in this? I tried to read this to learn where to go. Without this guidance, i went to top nav and typed in "SAML" and got no results, then i went to administration, and there is not an option for SAML... I then went to the docs, where after the "pre" section I'm presented with the path. Also noteworthy that you have that path setup as "Administration"->"Authentication"-> "SAML Configuration" but the actual tile is just "SAML".